### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SudharsanSundar**
Approach: Used basically identical hyperparams from my best training run of the TinyStories model, particularly a "constant" lr of 1e-3 with 1000 step warm-up. Found this worked well enough on first run and decided to stick with it! (And I didn't have a whole lot of time to try other things anyway ':D). I found the TS learning rate by more or less binary searching within the lr interval of 1e-1 to 1e-5, keeping warmup iterations constant and keeping the lr constant for the all training (aside from warmup). 

That being said, if I had (a lot) more time, something I would've *loved* to try would've been to "sort" OWT data by difficulty of the text (e.g. by Flesch reading score: https://readable.com/readability/flesch-reading-ease-flesch-kincaid-grade-level/) and then train the model starting on the easiest text and progressing toward harder text (and, e.g., only moving to harder text once loss decreases below some threshold). Very curious what kind of model this would result in, whether it'd be better on validation cross-entropy, what its "qualitative" behavior would be like, etc. compared to a model training on randomly shuffled OWT data (the default).

(Note: My model has 128 token context length (just happenstance, forgot to change this in the training args), so not sure if comparing my validation loss to (I'm assuming) 256 token context length models is apples to apples. 

That being said, smaller context length = harder task, right? Intuitively, smaller context length means less prior information to condition on and narrow down the distribution of the next token. But perhaps I'm missing something? 

("Proof": you can always "disregard" preceding tokens, so the longer context setting reduces to the small context setting, and, hence, a "perfectly rational machine" should do (weakly) better in longer contexts than shorter ones. Does this apply to LMs at all? Who knows : ) ) 

Either way, might be an interesting discussion (e.g. nuances about "misleading information/context", etc.) haha)

### CODE REVIEW

**Comment by nelson-liu** [comment]
> That being said, smaller context length = harder task, right? Intuitively, smaller context length means less prior information to condition on and narrow down the distribution of the next token. But perhaps I'm missing something?

Yeah, exactly.

thanks for the detailed writeup!
